modules/auxiliary/gather/coldfusion_pwd_props.rb

```diff
@@ -29,12 +29,6 @@ def initialize(info = {})
           'nebulus'
         ],
       'License'        => MSF_LICENSE,
-      'Actions'     =>
```

Comment:
Please update the module description!

Reply:
Oh yes indeed good catch. I'll do!

Reply:
Done in 0085b2c93e4f6a3a84355ad641bfd130ac61b91b